app/presenters/embassy_presenter.rb

```diff
@@ -42,6 +42,11 @@ def organisation
   end
 
   SPECIAL_CASES = {
+    "Central African Republic" => {
+      building: "Foreign and Commonwealth Office",
+      location: "the UK",
+      base_path: "https://www.gov.uk/government/organisations/foreign-commonwealth-office",
```

Comment:
Looks good - would it be worth removing the `https://www.gov.uk/` from the base path though to keep it consistent with the others?